Dashboard/static/styles/styles.css

```diff
@@ -0,0 +1,43 @@
+body {
```

Comment:
Here without doing modification to the body, we can introduce the container class of bootstrap?

Reply:
Sure